unit_tests/test_seed_selector.py

```diff
@@ -28,6 +28,10 @@ def ip_address(self):
         # Expected node name like : node1, node2, node3 ...
         return '127.0.0.%s' % self.name.replace('node', '')
 
+    @cached_property
```

Comment:
this isn't imported.

BTW I think you can make it a regular property

Reply:
Yes, lock of 'from functools import cached_property'

fixed.